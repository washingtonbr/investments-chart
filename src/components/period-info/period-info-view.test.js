import React from 'react';
import renderer from 'react-test-renderer';
import PeriodInfoView from './period-info-view';

const mockPeriodSelected = 'Desde o inÃ­cio';

describe('PeriodInfoView', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <PeriodInfoView
          periodSelected={mockPeriodSelected}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
