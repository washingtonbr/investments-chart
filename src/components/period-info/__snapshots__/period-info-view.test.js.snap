// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeriodInfoView renders correctly 1`] = `
<div
  className="sc-AxjAm eSpbeK"
>
  Você está vendo o período 
  <strong>
    Desde o início
  </strong>
  .
</div>
`;
